packages/legend-graph/src/graph-manager/protocol/pure/v1/transformation/pureGraph/to/helpers/V1_ValueSpecificationBuilderHelper.ts

```diff
@@ -728,12 +734,35 @@ const createThisVariableForClass = (
   return _var;
 };
 
```

Comment:
let's rename this to `findMappingLocalProperty` and move it to `DSL_Mapping_Helper.ts`

Reply:
@gayathrir11 this seems somewhat shaky to me, why do we find the local property mapping from a list of mappings? what are we actually trying to do here?

Reply:
I have made all the suggested changes